docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,6 +197,16 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
+* - `iceberg.object-store.enabled`
+  - Set to `true` to enable Iceberg's [object store file layout](https://iceberg.apache.org/docs/latest/aws/#object-store-file-layout). 
+    Enabling the object store file layout appends a deterministic hash directly 
+    after the data write path, configured via `iceberg.data-location`, to ensure
+    data files are equally distributed across multiple paths in a cloud storage 
+    service to avoid request throttling.
+  - `false`
+* - `iceberg.data-location`
+  - Sets the path that data files will be written to.
+  - table location + /data
```

Comment:
Ah, my mistake. I did mean to add this globally so it doesn't necessarily need to be configured per table. I'll add this in

Reply:
@ebyhr actually, looking at it I have this property set up in the same way `iceberg.file-format` is set up so it should be configurable at the catalog level already.

Reply:
`iceberg.file-format` is unrelated. I'm asking the scenario when we want to set it globally. 
Please remove this property if you are not sure. We can add it in follow-up.

Reply:
Ah, gotcha.  I thought you were asking if it's possible to set it globally.

The use case for being able to set it globally would be if you want every table to be created with the object store file layout to preemptively avoid running into rate limiting. Without being able to set it globally, you would have to manually update tables that run into rate limiting which can be time consuming and prone to missing some tables that might run into the issue in the future.

Reply:
Setting it globally would also be helpful if you want to have all of your tables follow the same setup, instead of some using the object store file layout and some using the default

Reply:
I'm asking about `iceberg.data-location`, not `iceberg.object-store.enabled`. 

Enabling `iceberg.object-store.enabled` globally makes sense, but `iceberg.data-location` doesn't in my opinion. Let me remove `iceberg.data-location` from this PR.

Reply:
I think having `iceberg.data-location` at the catalog level would still be best, for similar reasons. Being able to set all tables to store their data files using object storage at the root of an S3 bucket for example